lib/rocket_chat/messages/room.rb

```diff
@@ -185,6 +185,23 @@ def invite(room_id: nil, name: nil, user_id: nil, username: nil)
       end
 
       #
+      # *.kick REST API
+      # @param [String] room_id Rocket.Chat room id
+      # @param [String] name Rocket.Chat room name (coming soon)
+      # @param [String] user_id Rocket.Chat user id
+      # @param [String] username Rocket.Chat username
+      # @return [Boolean]
+      # @raise [HTTPError, StatusError]
+      #
+      def kick(room_id: nil, name:nil, user_id: nil, username: nil)
```

Comment:
Hmm, actually looks like rubocop has an issue with this line. Needs a space between `name:` and `nil`. You'll need to rebase off master to pick up some other rubocop fixes i've merged